main.tex

```diff
@@ -18,6 +18,33 @@
 \begin{document}
 
 \maketitle
+\subsubsection {Facebook}
+Eine Studie von \textcite{empowermentThroughFacebook}, welche zwischen 2011 und 2012 in New Delhi durchgeführt wurde, zeigt, dass Facebook zu der Zeit eine große Rolle in der Gesellschaftsentwicklung spielte. 
+\medskip
+Da zu 2012 eine Regulierung von bis zu 200 SMS pro Tag veröffentlicht wurde(Press Information Bureau,
+Government of India, 2012), stieg das Interesse von Facebook zu der Zeit stark an,
```

Comment:
`Press Information Bureau,
Government of India, 2012` bitte als richtige Quelle